learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,96 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
+## Prerequisites
+
++ [docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
++ you [log in](https://docs.docker.com/engine/reference/commandline/login/) your docker with [docker hub]((https://hub.docker.com/)) account
++ you have and know your **public IP** address
++ you have **a bitcoin and a litecoin address** for receiving mining reward
++ you may need to setup your network to allow other bitmark nodes to communicate with your bitmark node. You can check more information on "Important Notice On Network" below.
+
+## Installation
+
+### Step 0: Preparation
++ Have your **Public IP** ready
++ Make sure following ports are accessible from internet
+    + port	**2136** for connecting to other peer bitmarkd nodes
+    + port	**2135** for publishing blockchain events
+    + port 	**2130** for Bitmark node RPC server
+
+You can check more information on "Important Notice On Network" below.
+
+
+### Step 1: Download and Run Script
+
+Downloaded script will help you to manage bitmark node container. The  installation of the script will pull down a latest bitmark node container image from docker hub and then start a bitmark node container for you.  To check what functions the script supports and how to run it, simply run ``` bash install-node-linux-mac.sh```.  or ```bitmarkNode-HyperV.bat```.
+
+
++  Linux and mac users
+    + [download scripts for linux and mac](https://bitmark-node-docker-scripts.s3-ap-northeast-1.amazonaws.com/install-node-linux-mac.sh)
+    + execute bash install-node-linux-mac.sh [Your Public IP]. When run the script without providing [Your Public IP], the script will find your IP from Internet
+
+        ```bash install-node-linux-mac.sh 123.123.123.123```
```

Comment:
A suggestion about the format:

- Run the downloaded script:
   ```shell
      $ bash install-node-linux-mac.sh <Your Public IP>
   ````
    > **Note:** In case the Public IP isn't provided, the script finds the IP from the Internet.

Reply:
@vienlt  do you think it is a good idea to tell the user we will find the IP for them and that IP is not workable in many cases ?

Reply:
@shannona  Can you give us some opinion?

Reply:
@pieceofr : I just change the format of the instructions without changing any content. What is the concern here?

Reply:
@vienlt  Got it. Now No concern